cogs/automation/intervally/daily_auto.py

```diff
@@ -1,6 +1,7 @@
 import asyncio
```

Comment:
Not related to the PR but `asyncio` imported but unused :)
Can either be fixed in this PR or a future "Fix lint diagnostics" PR